### PR TITLE
git'n a sticker

### PR DESCRIPTION
**Pull request by Priyapandeyyy**
💡 Pull Request for WebMasterLog 💡

>  Issue Title: Text-to-speech converter using HTML, Css and JS

- Name of the project: Text-to-speech converter
- Category: Vanilla Js
- Name: Priya Pandey
- GitHub ID: https://github.com/Priyapandeyyy
- Identify yourself: GSSOC Contributor

Closes: #135 

> Describe the add-ons or changes you've made 📃

I have added Text-to-speech Converter to the Vanilla Js Projects. This project has an attractive UI and also simple interface so that users face no difficulty in using the Converter

### Type of change ☑️

- [x ] New feature (non-breaking change which adds functionality)
- [x ] This change requires a documentation update

### How Has This Been Tested? ⚙️
Yes, this project has been tested well. 

> Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my code.
- [x] My changes generate no new warnings.

> SCREEN RECORDING AND SCREENSHOTS ☑️

Screenshot 2024-05-14 125149

https://github.com/Avdhesh-Varshney/WebMasterLog/assets/132647631/8c509a40-114e-4678-92d5-1e55aaf6774a

### CODE REVIEW

File: package-lock.json
Comment (Avdhesh-Varshney) [inline package-lock.json:1]: Irrelevant changes.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@Priyapandeyyy this pr is totally wrong.
- No proper folder structure. Keep the used images into `Images` directory.
- Folder name will not have spaces already mention in guidelines.
- And i have told you to put this project inside the `Vanilla-JS-Projects` directory. Still you kept it in `Front-end-Projects`.

Recreate this PR and next time properly read the guidelines.

**Comment by Priyapandeyyy** [comment]
@Avdhesh-Varshney Hii sorry I didn't read the guidelines properly. Can you allow me to PR again please

**Comment by Avdhesh-Varshney** [comment]
> @Avdhesh-Varshney Hii sorry I didn't read the guidelines properly. Can you allow me to PR again please

@Priyapandeyyy No requirement for asking, until issue is assigned to you. You can create another PR ok.
But next time follow the guidelines carefully.
For further communication and queries, join this discord server https://discord.gg/tSqtvHUJzE

**Pull request closed without merging**